upstairs/src/volume.rs

```diff
@@ -1441,11 +1539,11 @@ impl Volume {
                 "sub_volume extent_count mismatch"
             )
         }
-        // The original generation number should always
-        // be lower than the new.  This could almost be
-        // a panic, as if they are the same, something
-        // has gone wrong.
-        if o_sv_gen >= n_sv_gen {
+
+        // The original generation number should always be lower than the new.
+        // This could almost be a panic, as if they are the same, something has
```

Comment:
I would just take out the sentence "This could almost be ...".  I know Its not one you
added.

Reply:
agree, nuked in 89a5393c